src/regfox/payloads/regfox_api_add_note.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * @return {*} a body object that can add a note to a registration.
+ *
+ * @param {string} message that should be seen as a note
+ * @param {string} id of registration (**not** the RegistrationId)
+ */
+const buildAddNoteBody = (message, id) => {
+  // I have no idea what these values mean and can't find any documentation on them.
+  const noteType = 5;
+  const createdBy = 144247; // Seems we can set this to any value?
```

Comment:
Maybe the UserID of the person creating the note?

Reply:
Not sure how to get that at the moment. But I'll make an action item to look into it!